test/drenv/__main__.py

```diff
@@ -37,15 +38,26 @@ def main():
         env = envfile.load(f, name_prefix=args.name_prefix)
 
     func = globals()[CMD_PREFIX + args.command]
-    func(env)
+
+    if args.skip_tests and "cmd_start" in str(func.__name__):
+        func(env, 1)
+    else:
+        func(env)
 
 
-def cmd_start(env):
+def cmd_start(env, skiptest=0):
     start = time.monotonic()
     logging.info("[%s] Starting environment", env["name"])
     # Delaying `minikube start` ensures cluster start order.
-    execute(start_cluster, env["profiles"], delay=1)
-    execute(run_worker, env["workers"], hooks=["start", "test"])
+
+    if skiptest:
+        hooks = ["start"]
+        execute(start_cluster, env["profiles"], skiptest=1, delay=1)
+    else:
+        hooks = ["start", "test"]
+        execute(start_cluster, env["profiles"], delay=1)
+    execute(run_worker, env["workers"], hooks)
```

Comment:
Here we need to:
- use the right hooks to run the worker
- pass te skip option down to start_cluster so it can build the right hooks list again

It is sad that we need to build the hook list twice. I think we can clean this up
by building the hooks once here like this:

    hooks = ["start", "test"] if args.run_tests else ["start"]

And pass down the hooks to the start_cluster, instead of the new option:

    execute(start_cluster, env["profiles"], delay=1, hooks=hooks)                                                                
    execute(run_worker, env["workers"], hooks=hooks)

To make it work, we can add a hook= argument o start_cluster:

    def start_cluster(profile, hooks=("start", "test"), **options):
        ...
        execute(run_worker, profile["workers"], hooks=hooks)

This way we manage the hook in a single location and make the code easier
to understand.